packages/web-app-files/src/mixins/spaces/actions/editReadmeContent.js

```diff
@@ -0,0 +1,88 @@
+import { mapActions, mapMutations, mapState } from 'vuex'
+
+export default {
+  data: function () {
+    return {
+      $_editReadmeContent_modalOpen: false,
+      $_editReadmeContent_content: ''
+    }
+  },
+  computed: {
+    ...mapState('Files', ['currentFolder']),
+    $_editReadmeContent_items() {
+      return [
+        {
+          name: 'editReadmeContent',
+          icon: 'markdown',
+          label: () => {
+            return this.$gettext('Change description')
+          },
+          handler: this.$_editReadmeContent_trigger,
+          isEnabled: ({ resources }) => {
+            if (!resources[0].spaceReadmeData) {
```

Comment:
is there always a resource in the resources array? else we could get spaceReadmeData of undefined

Reply:
good point, the if conditions are in wrong order